Gemfile

```diff
@@ -3,10 +3,10 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.3.0"
+ruby "3.3.2"
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.1.2"
+gem "rails", "~> 7.1.3.3"
```

Comment:
I was about to go for rails edge, but Bullet does not support the 8.0 alpha versions of ActiveRecord and such. I backed because I will be very reliant on Bullet for the next tasks (all tasks, to be honest).